nav/stepper/stepper_constants.py

```diff
@@ -20,13 +20,38 @@
 FL_step = 24
 FL_dir = 23
 
+<<<<<<< HEAD
+# Direction constants
+FWD = True
+REV = False
+
+# Clockwise and Counter-clockwise constants 
+CW = 1 			                 
+CCW = 0
+
+# whole, 1/2, 1/4, and 1/8 steps per revolution constants.
+# steps per revolution is obtained by (360/1.8) * denominator. 
+# ie: (360/1.8) * 2 = 400 for half steps per revolution
+STEPS_PER_REVOLUTION = 200
```

Comment:
Let's have this is its own dict as they are all related. So,
```
STEPS_PER_REVOLUTION = {'HALF_STEP': 400, 'QUARTER_STEP': 800 ...}
```
It will be easier to move around a dict than four variables individually.